config/prod/prow/jobs/custom/branchprotector.yaml

```diff
@@ -15,34 +15,35 @@
 # branchprotector is a tool implemented by k8s sig-testing.
 # It can configures github branch protection rules according to the specified policy in a YAML file.
 
-presubmits:
-  knative/test-infra:
-  - name: pull-knative-test-infra-branchprotector
-    agent: kubernetes
-    decorate: true
-    path_alias: knative.dev/test-infra
-    run_if_changed: "^config/branch_protector/rules.yaml$"
-    cluster: "prow-trusted"
-    branches:
-    - "master"
-    spec:
-      containers:
-      - name: branchprotector
-        image: gcr.io/k8s-prow/branchprotector:v20201103-bb494c0354
-        command:
-        - /app/prow/cmd/branchprotector/app.binary
-        args:
-        - --config-path=config/branch_protector/rules.yaml
-        - --github-token-path=/etc/github/token
-        - --confirm=false
-        volumeMounts:
-        - name: oauth
-          mountPath: /etc/github
-          readOnly: true
-      volumes:
-      - name: oauth
-        secret:
-          secretName: github-token-for-branchprotector
+# Do not enable presubmit tests for now since it's taking extra long time.
+#presubmits:
```

Comment:
Is there an issue to fix this?

Reply:
Thanks for the reminder, just created the issue - https://github.com/knative/test-infra/issues/2559